### DISCUSSION TITLE
Truncated description of sub-commands in auto-generated documentation.

### DISCUSSION THREAD

**Discussion by jhrcook** [tiangolo/typer-cli#141] [Questions] [answered]
The sub-commands for a command are listed under "**Commands:**" followed by their description. For long descriptions of the sub-commands, they are truncated and an ellipses ("...") is appended. Is there are way to not have the ellipses? I understand that we wouldn't want to have a multi-lined description for the sub-commands in this location, but maybe truncating at the first line would be a good option?

**Comment by tiangolo** [comment, marked as answer]
Hello! This repo, `typer-cli`, is now integrated into `typer` itself.

And this question would be more relevant to Typer directly.

Please create a new question there, following the instructions: https://github.com/tiangolo/typer/discussions/new?category=questions


And please create a small example that I can copy paste to see your use case, maybe also a screenshot to see how it looks on your side. :nerd_face:
